### DISCUSSION TITLE
Pulling code from geeksforgeeks.com

### DISCUSSION THREAD

**Discussion by username5484** [community/community#6826] [Copilot Conversations]
I was testing copilot's ability to produce classes and functions in python, and was perplexed when asking it to produce a graph class. As it produced an example from the site geeksforgeeks, going so far as to name the author from the site as well at the end of the file in comments. 
# https://www.geeksforgeeks.org/dijkstras-shortest-path-algorithm-using-priority_queue-stl/

**Comment by MrTeale** [comment]
I'm not one of the employees but a curious user of CoPilot. Have you got any steps I could use to replicate this?
